src/Routers/SchemasRouter.js

```diff
@@ -38,11 +38,10 @@ function mongoSchemaToSchemaAPIResponse(schema) {
 }
 
 function getAllSchemas(req) {
-  return req.config.database.collection('_SCHEMA')
-    .then(coll => coll.find({}).toArray())
-    .then(schemas => ({response: {
-      results: schemas.map(mongoSchemaToSchemaAPIResponse)
-    }}));
+  return req.config.database.adaptiveCollection('_SCHEMA')
+    .then(collection => collection.find({}))
+    .then(schemas => schemas.map(mongoSchemaToSchemaAPIResponse))
+    .then(schemas => ({ response: { results: schemas }}));
```

Comment:
Putting this in a separate `then` is actually marginally slower than not doing so, because each `then` callback happens in a separate turn of the event loop. Very minor difference, but I thought you might want to know (if you end up contributing to the dashboard at all, avoiding extraneous `then`s is a bigger deal because doing too many can cause a re-render on each turn of the event loop)